miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryDilutionRestController.java

```diff
@@ -139,4 +140,11 @@ public DilutionDto updateDilution(@PathVariable Long dilutionId, @RequestBody Di
     return jQueryBackend.get(request, response, null, PaginationFilter.pool(poolId));
   }
 
+  @RequestMapping(value = "query", method = RequestMethod.POST, produces = { "application/json" })
+  @ResponseBody
+  public List<DilutionDto> getSamplesInBulk(@RequestBody List<String> names, HttpServletRequest request, HttpServletResponse response,
+      UriComponentsBuilder uriBuilder) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)